discoverd2/health/monitor.go

```diff
@@ -0,0 +1,114 @@
+package health
+
+import (
+	"time"
+
+	"github.com/flynn/flynn/pkg/stream"
+)
+
+type MonitorConfig struct {
+	// StartInterval is the check interval to use when waiting for the service
+	// to transition from created -> up. It defaults to 100ms.
+	StartInterval time.Duration
```

Comment:
So if an app requires longer to start than 100ms, it will be marked as down until the next check 2 seconds later? Seems a bit penalising, but I can't think of a better solutio n.